### PR TITLE
added @comment.comment ( comment info )

### LINKED ISSUE #4503: Add comment text to moderator email
I am now receiving comment moderation emails - awesome work @ValentinaTironi @gauravano and others!

image

I think we need to add the comment text itself into the template so that people can skim the comment before approving it:

https://github.com/publiclab/plots2/blob/322b7561e81fd207fcb75a7c0e48940ecf891dad/app/views/admin_mailer/notify_comment_moderators.html.erb

What do you think? Thanks!

Also, i noted that when i clicked "Approve" the comment didn't seem to appear until I reloaded the page. I think we may (separately) need to add `"?_" + Time.now.to_s` to the end of the route to clear the cache in the browser.

### PR DESCRIPTION
**Pull request by Dsxv**
Also added some inline styles to Approve and Spam .

Fixes #4503 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
oh cool -- would you mind posting a screenshot? Thanks!!

**Comment by oorjitchowdhary** [comment]
Also, add the issue number in the PR. Thanks

**Comment by Dsxv** [comment]
screenshot from 2019-01-03 23-47-48

> The image is  UI look of  `notify_comment_moderators.html.erb` with help of another rails app  and is not the actual mail from the mailer of plots2 , ( I particularly not know how to preview mails without all the hassle of installation and setting up) 

> 
If anyone could Help me with it , I'd certainly appreciate it! 
Thanks for Help!

File: app/views/admin_mailer/notify_comment_moderators.html.erb [hunk 1]
```diff
@@ -9,14 +9,21 @@
...
 
-  <p>
-    <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/admin/publish_comment/<%= @comment.id %>">Approve</a>
+    <p> <b> Comment: </b> </p>
+      <div style = "padding:2vw; background-color:rgb(217, 217, 217); border-radius:5px;">
+        <%= @comment.comment %>
+      </div>
+            
+  <p style="text-align:center; padding: 5px;">
+    <a href="https://<%= ActionMailer::Base.default_url_options[:host] %>/admin/publish_comment/<%= @comment.id %>"
```
Comment (gauravano) [inline app/views/admin_mailer/notify_comment_moderators.html.erb]: Please remove all the styles added so as to match other email templates. Thanks!

**Comment by gauravano** [comment]
Please mark the relevant checkboxes in the PR. Also, using a separate feature branch is a good practice so that your master branch remain clean.

**Comment by Dsxv** [comment]
Sure @gauravano ! I'll keep that in mind from now on ! Thanks for reviewing !

**Comment by gauravano** [comment]
Hi @Dsxv I tried your PR locally, but I had some design suggestions:
comm_design

How about this? You can refer to code - https://github.com/gauravano/plots2/commit/1c3ed6b9ebdc245ae5d8805ad01a0b28d044e337

Thanks!

**Comment by gauravano** [comment]
Nice! Push your final changes please. Thanks!

**Comment by gauravano** [comment]
Cloing this PR as you have opened new PR at #4532. Thanks!

**Pull request closed without merging**